proposals/fate-operator-proposal.md

```diff
@@ -0,0 +1,220 @@
+
+-   [Background](#background)
+-   [Motivation](#motivation)
+-   [Goals](#goals)
+-   [Non-Goals](#non-goals)
+-   [Design](#design)
+    -   [Container images and deploying FATE cluster on
+        Kubernetes](#container-images-and-deploying-fate-cluster-on-kubernetes)
+    -   [Custom Resource Definition](#custom-resource-definition)
+        -   [Kubefate](#kubefate)
+        -   [FATECluster](#fatecluster)
+        -   [FateJob](#fatejob)
+    -   [Controller design](#controller-design)
+-   [Reference](#reference)
+
+_Status_
+* 2020-5-28 - Draft v2
+* 2020-5-14 – Draft v1
+
+## Background
+Federated machine learning (FML) is a machine learning setting where many clients (e.g. mobile devices or organizations) collaboratively train a model under the coordination of a central server while keeping the training data decentralized. Only the encrypted mediate parameters are exchanged between clients with MPC or homomorphic encryption.
+![Federated Machine Learning](diagrams/fate-operator-fl.png)
+
+FML has received significant interest recently, because of its effectiveness to solve data silos and data privacy preserving problems. Companies participated in federated machine learning include 4Paradigm, ANT Financial, Data Republic, Google, Huawei, Intel, JD.com, Microsoft, Nvidia, OpenMind, Pingan Technology, Sharemind, Tencent, VMware, Webank etc. 
```

Comment:
Companies doing research and building their technology solutions around federated machine learning include 4Paradigm, ANT Financial, Data Republic, Google, Huawei, Intel, JD.com, Microsoft, Nvidia, OpenMind, Pingan Technology, Sharemind, Tencent, VMware, Webank etc.

Reply:
ANT Financial -> Ant Financial

Reply:
@terrytangyuan BTW, is there any plan for Ant Financial to open source its internal FML framework?

Reply:
Not that I am aware of.